src/Raven.Client/Documents/Indexes/RollingIndex.cs

```diff
@@ -8,11 +8,13 @@ public class RollingIndex : IDynamicJson
     {
         public Dictionary<string, RollingIndexDeployment> ActiveDeployments = new Dictionary<string, RollingIndexDeployment>(StringComparer.OrdinalIgnoreCase);
 
+        public long RaftIndex;
```

Comment:
RaftCommandIndex (we use this name in other places)

Reply:
done